kong/runloop/plugins_iterator.lua

```diff
@@ -89,14 +128,15 @@ end
 local function get_next(self)
   local i = self.i + 1
 
-  local plugin = self.plugins_plan.loaded[i]
+  loaded_plugins = loaded_plugins or get_loaded_plugins()
```

Comment:
```suggestion
  if not loaded_plugins then
    loaded_plugins = get_loaded_plugins()
  end
```

Though, should it be in `self.iterator.loaded_plugins` to not check it on each iteration? Or why is this needef at all?

Reply:
Yes, I did this several times and forgot to undo this, thanks